### DISCUSSION TITLE
ROM location problems

### DISCUSSION THREAD

**Discussion by mowgli2k** [OnionUI/Onion#544] [Q&A]
Hi all - would really appreciate some help. 
Got my miyoo mini yesterday and immediately installed latest firmware and then onion OS. 
Mostly worked ok but 2 systems wouldn't show roms - when I pressed 'A", instead of the location being (SDCARD):/Roms/NES etc, it was showing (for example with SNES) :  (SDCARD):/Emu/SNES/../../Roms/SNES - I assumed the /../ bits were because the screen is too small to show full path. 
Today, I reinitialised the system, put main OS back on, then Onion OS - now NO games are not showing - they ALL show this :/Emu/../../ location and I cant see any. 
(If I run retroarch and browse to the folders, all games work fine).
Can someone either help me by telling me how you can change the location onion looks for roms in, or just give me the full location that Onion is trying to get to? 
THANK YOU.

**Comment by dwmccoy** [comment]
The ../../ is an actual change directory command.  In your example, it would indicate move up two directories to get to the ROOT folder of the SDCARD, then change into directory /Roms/SNES.  If you view the config.json file located in /EMU/SNES it will show you the ROM location path.  In short, almost all cores look for the roms in the /Roms/Game System folder off the ROOT of the SDCARD.  Examples would be like the following

NES roms would go here  /Roms/FC
SNES roms would go here  /Roms/SFC
Gameboy roms would go here  /Roms/GB
MAME roms would go here /Roms/ARCADE

Please refer to the wiki [https://github.com/jimgraygit/Onion/wiki/2.-Onion-Emulators-&-Ports](url) for more information
